jboss/container/wildfly/s2i/legacy/artifacts/opt/jboss/container/wildfly/s2i/galleon/provisioning/default_config/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
```

Comment:
it's time to update the date of your license templates :D

Reply:
Good point! Will fix.